iotedgedev/cli.py

```diff
@@ -67,12 +67,18 @@ def docker():
                        "Use \".\" as NAME to create in the current folder.")
 @click.argument("name",
                 required=True)
+@click.option("--runtime_tag",
+              "-rt",
```

Comment:
>-rt [](start = 15, length = 3)

Could you sync with Haitham on the namign of the parameter, he is enabling this for the simualtor and it may be more intuitive for users if they see similar parameter name.